README.md

```diff
@@ -65,3 +65,29 @@ You can set the file extension used by the UI to download a result from your fun
 * `environment`
 
 Set timeouts etc
+
+# OpenFaaS Template Store
+
+The Template Store are all the templates from OpenFaaS and OpenFaaS-Incubator organizations, along with user-defined templates which were approved by the community.
+
+## How to submit to the Template Store
+
+If you'd like to see your template in the official store follow these steps:
+
+1. Open issue with your template repository
+2. Check if the template is approved by the community
```

Comment:
'Await feedback on, and subsequent approval of, your proposal from the community.'

Reply:
Done ✔️